### PR TITLE
Require additional and updated testing packages for the CR test suite

### PR DESCRIPTION
**Pull request by nezaniel**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -40,9 +40,10 @@
     "neos/setup": "@dev",
     "flowpack/neos-frontendlogin": "@dev",
     "mikey179/vfsstream": "^1.6",
-    "phpunit/phpunit": "^9.0",
+    "behat/behat": "^3.0",
+    "phpunit/phpunit": "^10.0",
     "symfony/css-selector": "^2.0",
-    "vimeo/psalm": "~4.9.3",
+    "vimeo/psalm": "^5.0",
```
Comment (mhsdesign) [inline composer.json:46]: are we sure about this? - i the code compliant?

File: composer.json [hunk 2]
```diff
@@ -40,9 +40,10 @@
     "neos/setup": "@dev",
     "flowpack/neos-frontendlogin": "@dev",
     "mikey179/vfsstream": "^1.6",
-    "phpunit/phpunit": "^9.0",
+    "behat/behat": "^3.0",
+    "phpunit/phpunit": "^10.0",
```
Comment (mhsdesign) [inline composer.json:44]: dont we have like code that needs to be adjusted? ^^

**Review by mhsdesign — APPROVED** [review]
Fine by reading, but still have some questions ^^ - i guess if ci fails we need to revert xD

File: composer.json (hunk 1 above)
Comment (nezaniel) [reply composer.json:46]: Shouldn't be a problem (runs with phpunit 10 locally), but even if this fails for some reason it should be fixed asap in the dev collection

File: composer.json (hunk 2 above)
Comment (nezaniel) [reply composer.json:44]: see above

**Review by dlubitz — APPROVED** [review]

**Pull request merged**